scripts/config.py

```diff
@@ -0,0 +1,7 @@
+BASE_URL = "http://api.weatherapi.com/v1/history.json?key={}&q={}+united+states&dt={}"
+API_KEY = "a70a4e2736644cdcb9d85348202404"
```

Comment:
API keys must never be committed to version control. This must be removed and squashed before merging.